src/serialization.rs

```diff
@@ -363,6 +365,7 @@ pub mod audit {
                 version,
                 delta,
                 violation,
+                importable: if val.importable { None } else { Some(false) },
```

Comment:
I'm a bit concerned about this conversion because we lose whether an entry was explicitly set to `importable: true` in a round-trip serialization, which (in the future?) may have a different meaning than omission (e.g. we may have different rules about the default value when omitted). For instance, I was expecting `AuditEntry::importable` to be an `Option<bool>` and to see (at some point) something like
```
let entry: AuditEntry;
let is_importable = entry.importable.unwrap_or(!entry.kind.involves_git_revs());
```

But maybe you wanted to disallow ever importing git audits?

Reply:
The idea here is to be fully backwards compatible, meaning that no entry which was previously importable becomes non-importable after these changes. If the git entry was created before this patch is landed, it will not have `importable: false` specified on it, and so will be considered importable. Rather than making it so the default behaviour for git entries has been changed, this changes the code such that we specify `importable: false` for all new entries with git revisions.

Reply:
I see, that sounds reasonable.